orc8r/lib/go/initflag/initflag.go

```diff
@@ -10,58 +10,47 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-// package initflag initializes (parses) Go flag if needed, it allows the noise free use of golog & other packages
-// relying on flags being parsed
+
+// Package initflag parses a subset of flags, optionally redirecting output
+// to syslog.
 package initflag
 
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 )
 
-func init() {
-	flag.Var(
-		&syslogDest,
-		syslogFlag,
-		"Redirect stderr to syslog, optional syslog destination in network::address format (system default otherwise)")
+var (
+	flags          = flag.NewFlagSet("magma_tmp_flags", flag.ContinueOnError)
+	stdoutToStderr = flags.Bool("stdout2stderr", false, "Redirect stdout to stderr")
```

Comment:
Adding the flag to a local flagset would break applications calling the usual flag.Parse() if this flag is specified on command line, see: https://play.golang.org/p/OM04hvGvWjh for example